commands/lint.ts

```diff
@@ -73,7 +89,19 @@ export default class LintCommand extends BaseCommand {
 
   async lintRustFiles() {
     const dockerShellCommandExecutor = await this.dockerShellCommandExecutor("rust-tools");
-    await dockerShellCommandExecutor.exec(`find . -name '*.rs' | xargs rustfmt --edition "2021" --check`);
+    await dockerShellCommandExecutor.exec(`find . -name '*.rs' -exec rustfmt --edition "2021" --check -- {} +`);
+  }
+
+  async lintDockerFiles(tmpDirectory: string) {
+    const dockerShellCommandExecutor = await this.dockerShellCommandExecutor("docker-tools");
+    await dockerShellCommandExecutor.exec(`cp -r dockerfiles/ ${tmpDirectory}/`);
+    await dockerShellCommandExecutor.exec(`sed -i "/^COPY /s/--exclude=[^ ]*//g" ${tmpDirectory}/*.Dockerfile`);
+    await dockerShellCommandExecutor.exec(`hadolint --ignore DL3059 ${tmpDirectory}/*.Dockerfile`);
+  }
+  
+  async cleanupAfterLintDockerFiles(tmpDirectory: string) {
+    const dockerShellCommandExecutor = await this.dockerShellCommandExecutor("docker-tools");
+    await dockerShellCommandExecutor.exec(`rm -rf ${tmpDirectory}`);
```

Comment:
We don't need dockerShellCommandExecutor to remove files here btw, regular `ShellCommandExecutor` would do